runtime/queries/pkl/indents.scm

```diff
@@ -0,0 +1,32 @@
+; Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+;
+; Licensed under the Apache License, Version 2.0 (the "License");
+; you may not use this file except in compliance with the License.
+; You may obtain a copy of the License at
+;
+;     https://www.apache.org/licenses/LICENSE-2.0
+;
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS,
+; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+; See the License for the specific language governing permissions and
+; limitations under the License.
+
+; this definition is imprecise in that 
+; * any qualified or unqualified call to a method named "Regex" is considered a regex
+; * string delimiters are considered part of the regex
+[
+  (objectBody)
+  (classBody)
+  (ifExpr)
+  (mlStringLiteral) ; This isn't perfect; newlines are too indented but it's better than if omitted.
+] @indent
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "if"
+  "else"
+] @branch ; TODO: what does this query do?
```

Comment:
I believe that `@branch` is specific to nvim's indent system. We could remove the indent query for now and rely on the default behavior until someone wants to write Helix-specific ones

Reply:
Done, thanks for review.